src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">My name is {person.name}</h2>
+    {Object.prototype.hasOwnProperty.call(person, 'age') && (
+      <p className="Person__age">I am {person.age}</p>
+    )}
+    {!person.isMarried ? (
+      <p className="Person__partner">I am not married</p>
+    ) : person.sex === 'm' ? (
+      <p className="Person__partner">{person.partnerName} is my wife</p>
+    ) : (
+      <p className="Person__partner">{person.partnerName} is my husband</p>
+    )}
```

Comment:
[CODE STYLE] The ternary operator used in lines 7-13 is quite complex and can be hard to read. Consider moving each operand to a separate line or even better, create a separate variable for the condition. Refer to the checklist for the correct examples.